src/rpp/rpp/sources/timer.hpp

```diff
@@ -26,4 +26,19 @@ auto timer(rpp::schedulers::duration when, TScheduler&& scheduler)
 {
     return interval(when, std::forward<TScheduler>(scheduler)) | operators::take(1);
 }
+
+/**
+ * @brief Same as rpp::source::timer but using a time_point as delay instead of a duration.
+ *
+ * @param when time point when the value is emitted 
+ * @param scheduler the scheduler to use for scheduling the items
+ *
+ * @ingroup creational_operators
+ * @see https://reactivex.io/documentation/operators/timer.html
+ */
+template<schedulers::constraint::scheduler TScheduler>
+auto timer(rpp::schedulers::time_point when, TScheduler&& scheduler)
+{
+    return timer(when - rpp::schedulers::clock_type::now(), std::forward<TScheduler>(scheduler));
```

Comment:
i think, it is incorrect solution. Example:
```cpp
auto obs = timer(now + 1day);
wait_for(20h);
obs.subscribe();
```

Reply:
you should keep `when` inside observable and schedule to this timepoint at the moment of subscription

Reply:
done